### PR TITLE
New feature, issue report creation.

### PR DESCRIPTION
**Pull request by rajinder-yadav**
Take used to Augury Github issue page, create a new issues and pre-populate basic steps to follow.

augury-issue

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -1,17 +1,39 @@
...
-
-    var version = chrome.runtime.getManifest().version || "1.1.0";
     versionInstances[i].innerHTML = "(" + version + ")";
-
   }
 }
 
+function generateGithubIssuesBodyText() {
+  const date = (new Date()).toUTCString();
+  let body =`Augury: ${chrome.runtime.getManifest().version}
```
Comment (sumitarora) [inline popup.js]: should be const if not changing

Comment (stevenkampen) [inline popup.js]: @rajinder-yadav Also I think there is a space missing after the `=` sign.

Comment (rajinder-yadav) [inline popup.js]: Good catch guy, fixed.

**Comment by igor-ka** [comment]
@rajinder-yadav this is great!

**Pull request merged**